Swift templates compilation failed using standalone Sourcery app.

Branch: master.

Steps to reproduce:
- build Sourcery app through Xcode archive or `rake build`
- try to generate code with this app from a swift template

```
$ ../Sourcery/bin/sourcery --sources Sources --templates Templates --output Generated --verbose
Scanning sources...
Found 1 types.
Loading templates...
Loaded 1 templates.
Generating code...
Rendering template Templates/Sample.swifttemplate
/usr/bin/rsync "-av" "/Users/eugene/Dev/Contribution/Sourcery/bin/Sourcery.app/Contents/Frameworks/SourceryRuntime.framework" "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T"
/usr/bin/swiftc "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/main.swift" "-suppress-warnings" "-Onone" "-module-name" "main" "-target" "x86_64-apple-macosx10.10" "-F" "." "-o" "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/bin"
error: /var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/main.swift:2:8: error: no such module 'SourceryRuntime'
import SourceryRuntime
       ^
```

Possible issue:
Swift compiler can't import `SourceryRuntime` because `SourceryRuntime.framework` from app bundle does not contain swift module files.

I've already tried several similar workarounds. But they don't work.
```
$ ./Sourcery/bin/sourcery --sources Sources --templates Templates --output Generated --verbose
Scanning sources...
Found 1 types.
Loading templates...
Loaded 1 templates.
Generating code...
Rendering template Templates/Sample.swifttemplate
/usr/bin/rsync "-av" "/Users/eugene/Dev/Contribution/Sourcery/bin/Sourcery.app/Contents/Frameworks/SourceryRuntime.framework" "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T"
/usr/bin/swiftc "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/main.swift" "-suppress-warnings" "-Onone" "-module-name" "main" "-target" "x86_64-apple-macosx10.10" "-F" "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T" "-o" "/var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/bin"
error: /var/folders/qz/097sq7mn4wx5g_b1j_xj_55c0000gn/T/Sourcery.build/main.swift:2:8: error: no such module 'SourceryRuntime'
import SourceryRuntime
       ^
```
Looks like `SourceryRuntime.framework` from `build/Build/Products/` should be copied into app resources and then used to compile swift templates.

--- a/.sourcery.yml
+++ b/.sourcery.yml
@@ -1,6 +1,6 @@
 sources:
-  - SourceryFramework/Sources
+  - SourceryRuntime/Sources
 templates:
   - Sourcery/Templates
 output:
-  Sourcery/CodeGenerated
+  SourceryRuntime/Sources 

--- a/Sourcery.xcodeproj/project.pbxproj
+++ b/Sourcery.xcodeproj/project.pbxproj
@@ -51,7 +51,6 @@
 		09C6B0D41E11C1C500246CDF /* ClassSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = 09C6B0D31E11C1C500246CDF /* ClassSpec.swift */; };
 		58A8D89B0B5015671E6B526C /* Pods_SourceryTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6F74DA40979C0911C0573C7A /* Pods_SourceryTests.framework */; };
 		64E201AA1EAE8FBF00EAD8A2 /* SourceryRuntime.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 64E201A31EAE8FBF00EAD8A2 /* SourceryRuntime.framework */; };
-		64E201AB1EAE8FBF00EAD8A2 /* SourceryRuntime.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 64E201A31EAE8FBF00EAD8A2 /* SourceryRuntime.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		985E4D303A731C40BE326E5C /* Pods_Sourcery.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D7C9B21117B6C7EB409B9D49 /* Pods_Sourcery.framework */; };
 		C894A1B01E19326C0098327C /* SwiftTemplate.swift in Sources */ = {isa = PBXBuildFile; fileRef = C894A1AE1E19326C0098327C /* SwiftTemplate.swift */; };
 		C894A1B31E19328E0098327C /* SwiftTemplates in Resources */ = {isa = PBXBuildFile; fileRef = C894A1B21E19328E0098327C /* SwiftTemplates */; };
@@ -111,20 +110,6 @@
 			remoteInfo = Sourcery;
 		};
 /* End PBXContainerItemProxy section */
-
-/* Begin PBXCopyFilesBuildPhase section */
-		CD754CB41D85431F0082B512 /* Embed Frameworks */ = {
-			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = "";
-			dstSubfolderSpec = 10;
-			files = (
-				64E201AB1EAE8FBF00EAD8A2 /* SourceryRuntime.framework in Embed Frameworks */,
-			);
-			name = "Embed Frameworks";
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
 		0906124C1E2BC43500BA0937 /* FileParser + MethodsSpec.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "FileParser + MethodsSpec.swift"; sourceTree = "<group>"; };
@@ -617,7 +602,7 @@
 				CD754C891D853F000082B512 /* Resources */,
 				C7CDE8D445443BF7F4CF3AEB /* [CP] Embed Pods Frameworks */,
 				A7A107008E6572DA90B257F6 /* [CP] Copy Pods Resources */,
-				CD754CB41D85431F0082B512 /* Embed Frameworks */,
+				63F19A081ED109A90046D2EE /* ShellScript */,
 			);
 			buildRules = (
 			);
@@ -759,6 +744,19 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-SourceryTests/Pods-SourceryTests-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
+		};
+		63F19A081ED109A90046D2EE /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "/usr/bin/rsync -av \"${BUILT_PRODUCTS_DIR}/SourceryRuntime.framework\" \"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}\"";
 		};
 		8E96ACE81324E260FA5DCECB /* [CP] Check Pods Manifest.lock */ = {
 			isa = PBXShellScriptBuildPhase;

--- a/Sourcery/Generating/Template/Swift/SwiftTemplate.swift
+++ b/Sourcery/Generating/Template/Swift/SwiftTemplate.swift
@@ -162,7 +162,7 @@ class SwiftTemplate: Template {
                 "-Onone",
                 "-module-name", "main",
                 "-target", "x86_64-apple-macosx10.10",
-                "-F", ".",
+                "-F", buildDir.parent().description,
                 "-o", binaryFile.description
         ]
 

--- a/Sourcery/Generating/Template/Swift/SwiftTemplate.swift
+++ b/Sourcery/Generating/Template/Swift/SwiftTemplate.swift
@@ -121,8 +121,11 @@ class SwiftTemplate: Template {
     func render(types: Types, arguments: [String: NSObject]) throws -> String {
         let binaryPath: Path
 
-        if let cachePath = cachePath, let hash = code.sha256() {
-            binaryPath = cachePath + hash
+        if let cachePath = cachePath,
+            let hash = code.sha256(),
+            let hashPath = hash.addingPercentEncoding(withAllowedCharacters: CharacterSet.alphanumerics) {
+            
+            binaryPath = cachePath + hashPath
             if !binaryPath.exists {
                 try? cachePath.delete() // clear old cache
                 try cachePath.mkdir()

